mobile-widgets/qml/DivePlannerSetup.qml

```diff
@@ -11,11 +11,19 @@ Kirigami.ScrollablePage {
 	title: qsTr("Dive planner setup")
 
 	property string textLength: (PrefUnits.unit_system === "metric") ? qsTr("m/min") : qsTr("ft/min")
-//	property string textLength: PrefUnits.unit_system
 	onTextLengthChanged: {
-		console.log("--> New text << " + textLength )
+		console.log("---> STARTING to put new values")
 		spinAscrate75.value++
 		spinAscrate75.value--
+		spinAscrate50.value++
+		spinAscrate50.value--
+		spinAscratestops.value++
+		spinAscratestops.value--
+		spinAscratelast6m.value++
+		spinAscratelast6m.value--
+		spinDescrate.value++
+		spinDescrate.value--
+		console.log("---> new values READY")
```

Comment:
again, this doesn't look like the final code. If the increment/decrement pairs are needed, then we should have comments there explaining this

Reply:
Added a longer comment in one of the previous commits. Apart from the console.log (which is gone) this is the final code. At least until I find a better way to make the spinBox update its text.